message-index/messages/GHC-84077/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Type application without space
-summary:  Parsing error of type application.
+summary: A type application with @ does not have a space before.
```

Comment:
before -> in front of it?


As a non-native speaker, the word "before" has a temporal connotation for me.